### DISCUSSION TITLE
希望PC端也能支持VPN加速模式

### DISCUSSION THREAD

**Discussion by AkarinLiu** [BeyondDimension/SteamTools#1337] [Ideas]
现在手机端支持了VPN加速模式，PC端本就该支持了吧。

**Comment by yingmiait** [comment]
确实应该加入，host代理与系统代理方式对我无效

**Comment by ElemenTP** [comment]
Windows在Windows 10及以上才提供了系统的透明VPN接口，但必须是Appx打包格式的软件才可以使用，且微软商店不上架使用这个接口的应用，用户必须自签名安装，不太可能支持
